### ISSUE TITLE
Do not generate setter for private(set) field

### ISSUE THREAD

**Issue by rbeeger** [Brightify/Cuckoo#116]
A class I want Cuckoo to generate a mock for has a field like this
`private(set) var someField: SomeClass`

Cuckoo generates
```swift
    override var someField: SomeClass {
        get {
            return manager.getter("someField", original: observed.map { o in return { () ->SomeClass in o.someField } })
        }
        set {
            manager.setter("someField", value: newValue, original: observed != nil ? { self.observed?.someField = $0 } : nil)
        }
    }
```

The swift compiler rightly complains that the setter for `self.observed?.someField` is inaccessible.
Cuckoo should only generate a getter here. On a mock I would stub the getter and on a spy I wouldn't attempt to call the setter.

**Comment by TadeasKriz** [comment]
Hey, this is something we have fixed, but the branch is not ready yet. I rewrote the generator to use Stencil template file in order to improve readability. However I got quite busy few weeks behind me so I wasn't able to complete it and merge.

**Comment by rbeeger** [comment]
@TadeasKriz Thanks. Looking forward to the next release.

BTW: I've been wishing for useful reflection and mocking possibilities in swift since it was revealed. Cuckoo provides good mocking capabilities with todays swift. To you and the other contributors: Thanks for working on it.

**Comment by rbeeger** [comment]
Closing this as it was fixed in 0.9.0
